src/Spout/Reader/XLSX/Helper/CellValueFormatter.php

```diff
@@ -183,17 +182,25 @@ protected function formatNumericCellValue($nodeValue, $cellStyleId)
      */
     protected function formatExcelTimestampValue($nodeValue)
     {
-        $numDaysSince1970Jan1 = $nodeValue - self::NUM_DAYS_FROM_JAN_1_1900_TO_JAN_1_1970;
+        if (ceil($nodeValue) > self::ERRONEOUS_EXCEL_LEAP_YEAR_DAY) {
+            --$nodeValue;
+        }
 
-        // Fix for the erroneous leap year in Excel
-        if ($nodeValue < self::ERRONEOUS_EXCEL_LEAP_YEAR_DAY) {
-            $numDaysSince1970Jan1++;
+        if ($nodeValue < 1) {
+            return null;
         }
 
-        $unixTimestamp = round($numDaysSince1970Jan1 * self::NUM_SECONDS_IN_ONE_DAY);
+        // Do not use any unix timestamps for calculation to prevent
+        // issues with numbers exceeding 2^31.
+
+        $secondsRemainder = fmod($nodeValue, 1) * self::NUM_SECONDS_IN_ONE_DAY;
+        $secondsRemainder = round($secondsRemainder);
```

Comment:
can you specify the second param to be 0, even though it's the default value? it makes it more explicit that an int value is being returned